doc/input-output-reference/src/overview/group-python-plugins.tex

```diff
@@ -50,9 +51,8 @@ \subsection{Python Plugin}
 from pyenergyplus.plugin import EnergyPlusPlugin
 
 class DummyPlugin(EnergyPlusPlugin):
-
-def on_end_of_zone_timestep_before_zone_reporting(self):
-return 0
+  def on_end_of_zone_timestep_before_zone_reporting(self, state):
```

Comment:
Updated entire Python Plugin section for new state argument and updated workflow information.